src/endpoints/todoEndpoints.ts

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+import {Tasks} from "../models/ArrayModel"
+export default [
+    
+    // Get all
+    {
+        method: 'GET',
+        path: '/tasks',
+        handler: (
+            request: any, 
+            h: any) => 
```

Comment:
Part of the advantage of using typescript is being able to strongly type variables. Hapi has decent typescript support you need to do some research regarding using Hapi TypeScript types